custom_components/ha_washdata/frontend.py

```
@@ -379,23 +373,66 @@ async def async_register_panel(hass: HomeAssistant) -> bool:
+    if not hass.data.get(PANEL_REGISTERED_KEY) and PANEL_TASK_KEY not in hass.data:
         return
 
+    # Cancel and drain any in-flight registration task so it cannot complete
+    # and set PANEL_REGISTERED_KEY after we clear it below.
+    task: asyncio.Task[bool] | None = hass.data.pop(PANEL_TASK_KEY, None)
+    if task is not None and not task.done():
+        task.cancel()
+        try:
+            await task
+        except (asyncio.CancelledError, Exception):  # pylint: disable=broad-exception-caught
+            pass
```

Ruff (0.15.21)

[warning] Use `contextlib.suppress(asyncio.CancelledError, Exception)` instead of `try`-`except`-`pass`

(SIM105)